readme.md

```diff
@@ -155,12 +155,6 @@ Fires everytime new content is about to be loaded to the DOM.
 ```
 Handle loading finish.
 
-#### `PillOptions.onUnmounting()`
-```
-(page:Page, url:URL, element:HTMLElement) -> void
-```
-Fires everytime content is about to be removed from the DOM.
-
```

Comment:
Seems like you occasionally removed the documentation section. Please, return it back.

Reply:
Sure thing.  My apologies